src/panels/lovelace/editor/config-elements/hui-vacuum-commands-tile-extra-editor.ts

```diff
@@ -0,0 +1,104 @@
+import { html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import type { LocalizeFunc } from "../../../../common/translations/localize";
+import "../../../../components/ha-form/ha-form";
+import type { SchemaUnion } from "../../../../components/ha-form/types";
+import type { HomeAssistant } from "../../../../types";
+import {
+  VacuumCommandsTileExtraConfig,
+  VACUUM_COMMANDS,
+} from "../../tile-extra/types";
+import type { LovelaceTileExtraEditor } from "../../types";
+
+@customElement("hui-vacuum-commands-tile-extra-editor")
+export class HuiVacuumCommandsTileExtraEditor
+  extends LitElement
+  implements LovelaceTileExtraEditor
+{
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @state() private _config?: VacuumCommandsTileExtraConfig;
+
+  public setConfig(config: VacuumCommandsTileExtraConfig): void {
+    this._config = config;
+  }
+
+  private _schema = memoizeOne(
+    (localize: LocalizeFunc) =>
+      [
+        {
+          name: "commands",
+          selector: {
+            select: {
+              multiple: true,
+              options: VACUUM_COMMANDS.map((command) => ({
```

Comment:
Should we check if the chosen entity supports those commands, and only show the supported ones?

Reply:
I added a "not compatible" label to the commands. User can unselect it if it's not compatible but not select it.

![image](https://user-images.githubusercontent.com/5878303/203979103-c2f23d30-9cb6-4708-811c-6eeb064c0497.png)